etna/core/serializers/images.py

```diff
@@ -0,0 +1,121 @@
+from rest_framework.serializers import Serializer
+
+
+class ImageSerializer(Serializer):
+    """
+    This ImageSerializer was created to improve the `ImageRenditionField` that
+    comes as part of the Wagtail API. This serializer allows us to create a
+    JPEG and WEBP rendition in one field, rather than having to have multiple fields.
+
+    rendition_size defaults to `fill-600x400`, but can be specified
+    when the serializer is used, e.g:
+    ImageSerializer(rendition_size="original")
+
+    jpeg_quality and webp_quality default to 60 and 80 respectively,
+    and can be specified in the same way as rendition_size.
+
+    The source of the image can also be set in the serializer, e.g:
+    APIField("image_large", serializer=ImageSerializer(rendition_size="fill-900x900", source="image"))
+
+    This will come back as `image_large` in the API response.
+    """
+
+    def __init__(
+        self,
+        rendition_size="fill-600x400",
+        jpeg_quality=60,
+        webp_quality=80,
+        *args,
+        **kwargs,
+    ):
+        self.jpeg_quality = jpeg_quality
+        self.webp_quality = webp_quality
+        self.rendition_size = rendition_size
+        super().__init__(*args, **kwargs)
+
+    def to_representation(self, value):
+        if value:
+            jpeg_image = value.get_rendition(
+                f"{self.rendition_size}|format-jpeg|jpegquality-{self.jpeg_quality}"
+            )
+            webp_image = value.get_rendition(
+                f"{self.rendition_size}|format-webp|webpquality-{self.webp_quality}"
+            )
+
+            return {
+                "id": value.id,
+                "title": value.title,
+                "image_jpeg": {
```

Comment:
Please could we add the `alt` attribute to all images?

Reply:
...but outside of the `image_jpeg` or `image_webp` - alongside the `id` and `title`

Reply:
I'm fine with it being in each of the `image_jpeg` and `image_webp` then - just trying to reduce duplication 👍🏼

Reply:
Our images don't have any alt text, only if they're used in the page, then we add some at that level rather than at image level - so if an image is used it'll be like this:
![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/fc7b18bc-32d1-40fb-aee4-82092e7d5714)
`caption` and `alt_text` are part of the `ImageBlock` StreamField, so we can't add them to the `image` part because they're like separate fields, if that makes sense 😅